addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -114,6 +130,18 @@ if (_id != -1) then {
                 GVAR(oldUnit) = _player;
             };
 
+            private _data = group _player;
```

Comment:
Why re-private `_data` ? Line 151 is not doing that. In the old version only the first `_data` assignment was privated